lab1.py

```diff
@@ -0,0 +1,59 @@
+
+
+# Task1
```

Comment:
The idea of this task is to display all prime numbers within a range specified by the user. However, this is fine.
Good job Sara 👍🏽